src/Mechanic/ProjectCracker.fs

```diff
@@ -0,0 +1,154 @@
+/// Copied from https://github.com/fable-compiler/Fable/blob/f8457dc6ff3208af85166a14da0feb520927bc55/src/dotnet/dotnet-fable/ProjectCoreCracker.fs
```

Comment:
/cc @Krzysztof-Cieslak @enricosada @alfonsogarciacaro 
what's the best way to include projectcracker here?

Reply:
https://github.com/fsprojects/Mechanic/pull/77#issuecomment-372692877